app/enterprise/0.35-x/developer-portal/configuration/authentication/basic-auth.md

```diff
@@ -14,15 +14,21 @@ Basic Authentication for the Dev Portal can be enabled in three ways:
 - via the [the Kong configuration file](#enable-basic-auth-via-the-kong-conf)
 
 >**Warning** Enabling authentication in the Dev Portal requires use of the
-> Sessions plugin. Developers will not be able to login if this is not set 
-> properly.
+> Sessions plugin. Developers will not be able to login if this is not set
+> properly. More information about [Sessions in the Dev Portal](/enterprise/{{page.kong_version}}/developer-portal/configuration/authentication/sessions)
 
 ### Enable Portal Session Config
 
 In the the Kong configuration file set the `portal_session_conf` property:
 
 ```
-portal_session_conf={ "cookie_name": "portal_session", "secret": "super-secret", "cookie_secure": false, "storage": "kong" }
+portal_session_conf={ "cookie_name": "portal_session", "secret": "CHANGE_THIS", "storage": "kong" }
+```
+
+For testing purposes when using HTTP, config should include `"cookie_secure": false`:
```

Comment:
```suggestion
If using HTTP while testing, include `"cookie_secure": false` in the config:
```

Reply:
Since this is a duplicate of the doc in Kong Manager, would you recommend the same change there? I like your change, @levfishbluefish , just don't want to have a resulting double standard. LMK and I can PR the change separately.

Reply:
@h3xar0n yeah lets keep them consistent